go/registry/api/api.go

```diff
@@ -281,6 +284,30 @@ type NodeEvent struct {
 	IsRegistration bool
 }
 
+// RuntimeEvent signifies new runtime registration.
+type RuntimeEvent struct {
+	Runtime *Runtime
```

Comment:
Add explicit serialization field annotations to all event structures (also pre-existing ones are missing these). For example here it would be something like:
```suggestion
	Runtime *Runtime `json:"runtime"`
```